pyproject.toml

```diff
@@ -42,7 +43,7 @@ wheel.packages = []
 
 [tool.black]
 line-length = 88
-target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311', 'py312']
```

Comment:
Do we want to ditch 3.7 here? Numpy stopped supporting it in Dec'21
https://endoflife.date/numpy

Reply:
See also in other python versions.

Reply:
I don't. We don't really depend on a particular version of numpy, so there's no real need to deprecate, as long as it is recentish. Plus, Google Colab is still stuck on 3.7, which is/was a goto for some to get started with Notebooks. As long as it works, I'd prefer to keep that option open. I'm guess next Arbor release things will be different anyhow.

Reply:
Colab -- while true -- is a good reason not to.